BasalDose/BasalDose.c

```diff
@@ -94,10 +95,18 @@ void BasalDose_DoseInject(void)
 void BasalDose_RetractSyringe(void)
 {
 	LPC_TIM1->IR |= 1 << 0; // Clear MR0 interrupt flag
-	LPC_GPIO1->FIOPIN ^= 1 << 28; // Toggle the LED
 	
 	BasalDose_DoseTimingDisable(); // Disable Timer0
 	
+	// Turn P1.28 LED on to indicate stepper motor is spinning (backward)
+	LPC_GPIO1->FIOSET |= 1 << 28;
+	
+	/*
```

Comment:
Two on the top and bottom or just top? i.e

/**
 * Comment here
 **/ 
or
/**
 * Comment here
 */